app/code/core/Mage/Widget/Model/Widget/Instance.php

```diff
@@ -229,7 +229,7 @@ public function getType()
      */
     protected function _prepareType()
     {
-        if (strpos($this->_getData('type'), '-') >= 0) {
+        if (strpos((string)$this->_getData('type'), '-') !== false) {
```

Comment:
It seems PHPStan doesn't like this change? Not sure why but this would resolve:

```suggestion
        if (str_contains((string)$this->_getData('type'), '-')) {
```

Reply:
Whats wrong with phpstan?

Quick test; https://phpstan.org/r/40139d78-42e0-4376-81a0-ef01d82f251c

Reply:
str_contains is php8 only

Reply:
I guess @colinmollenhour is talking about something else - we have `str_contains` polyfill.

Reply:
Ahh I see now, these lines just need to be removed since the error was actually fixed: https://github.com/OpenMage/magento-lts/blob/1.9.4.x/phpstan.dist.baseline.neon#L7169